plugins/org.eclipse.fordiac.ide.model/src/org/eclipse/fordiac/ide/model/dataimport/CommonElementImporter.java

```diff
@@ -420,21 +422,44 @@ protected void readCommentAttribute(final INamedElement namedElement) {
 		}
 	}
 
+	private boolean hasXmlAttribute(final String localName) {
+		if (reader.getEventType() != XMLStreamConstants.START_ELEMENT) {
+			return false;
+		}
+		for (int i = 0; i < reader.getAttributeCount(); i++) {
+			if (localName.equals(reader.getAttributeLocalName(i))) {
+				final String v = reader.getAttributeValue(i);
+				return v != null && !v.isBlank();
+			}
+		}
+		return false;
+	}
+
 	protected void parseGenericAttributeNode(final ConfigurableObject confObject)
 			throws XMLStreamException, TypeImportException {
+		final String xmlTypeName = getAttributeValue(LibraryElementTags.TYPE_ATTRIBUTE);
```

Comment:
Please just name this `typeName`:
```suggestion
		final String typeName = getAttributeValue(LibraryElementTags.TYPE_ATTRIBUTE);
```
and remove line 460.